### PR TITLE
Pinan Update, Style info update

### PR DESCRIPTION
**Pull request by Neoxas**
Added basic information on the style, added links to pinan nidan

### CODE REVIEW

File: kata/pinan-nidan.kata [hunk 1]
```diff
@@ -5,11 +5,16 @@ name = {
 }
 
 info = {
-  ""
+  """
+  Pinan is the Chinese Pinyin notation of 平安, which means "peaceful and safe". This is often a simple kata taugh to emphasise the practicality of movements such as oi-zuki and agi-uki. All the kata's beginning with Pinan are based on an 'I' pattern. These were originally intended to be taugh to high school students as part of a national curriculum.
```
Comment (kinbiko) [inline kata/pinan-nidan.kata]: Pinan is not pinyin, that would be Píng'ān. The word "Pinan" comes from ピンアン which is the closest Japanese pronunciation of 平安 when pronounced in Chinese. (The Japanese reading is of course "heian").

Comment (kinbiko) [inline kata/pinan-nidan.kata]: ag*e*-uke.
kata's -> katas

Comment (kinbiko) [inline kata/pinan-nidan.kata]: I might be wrong in saying this so perhaps you could provide a reference...
I was of the impression that Naihanchi was the kata taught in high-school, and that the pinan katas were later introduced by Itosu Anko for younger children (elementary school?)

Comment (DB709) [inline kata/pinan-nidan.kata]: Nihanchi is a much older kata and was the original start kata in the shuri and tomori style syllabus. The pinans were definitely introduced into okinawan high schools ( not in Japan tho shorien Ryu has never been very present on the mainland with modern more Japanese styles more prevalent ). They were indeed created by itosu likely taking key moves from existing advanced kata.

Comment (kinbiko) [inline kata/pinan-nidan.kata]: See, I've got the [KARATE The Complete Kata by Kanazawa](https://www.amazon.co.uk/Karate-Complete-Kata-Hirokazu-Kanazawa/dp/1568365179) book which claims that the middle-school students in Okinawa would practice Naihanchi exclusively for 2-3 years, and that the Heian katas were conceived to provide easier means for mastering basic karate techniques.
Which to me seems to indicate that one would do the Pinans before Naihanchi, i.e. before high school.
I suggest removing the 'high/middle' whatever part of the sentence, and just stick with school in general.

Comment (DB709) [inline kata/pinan-nidan.kata]: Not to say Kanazawa is wrong, I would like to think he knows what he is talking about but from all the sources I've seen I haven't found any evidence of nihanchi being taught in schools after the creation of the pinans. The 2-3 years quoted is the amount of time quoted as how long all karate ka would spend on nihanchi before learning a second kata. I agree though that pinans where taught in elementary school I've just done a little research and sources seem to support their introduction initially to elementary schools.

Comment (Neoxas) [inline kata/pinan-nidan.kata]: I think if I just remove "High" like roger said then that will correct the issue. Might be worth putting another section about how it moved from nihanchi to Pinans?

File: kata/pinan-nidan.kata [hunk 2]
```diff
@@ -5,11 +5,15 @@ name = {
 }
 
 info = {
-  ""
+  """
+  The word "Pinan" comes from ピンアン which is the closest Japanese pronunciation of 平安 when pronounced in Chinese (The Japanese reading of 平安 is of course "heian") which means "peaceful and safe". This is often a simple kata taught to emphasise the practicality of movements such as oi-zuki and age-uki. All the katas beginning with Pinan are based on an 'I' pattern. These were originally intended to be taught to school students as part of a national curriculum.
```
Comment (kinbiko) [inline kata/pinan-nidan.kata]: uki -> uke

**Pull request closed without merging**